### ISSUE TITLE
SASS + CSS Injecting is not working since 2.24.0 version

### ISSUE THREAD

**Issue by Chmylov** [BrowserSync/browser-sync#1553]
### Issue details

Browsersync detects changes in css but it does not inject the new css since 2.24.0 version. You have to manually refresh the page to see the changes

### Steps to reproduce/test case

Just use example on https://browsersync.io/docs/gulp to reproduce this issue.

```
var gulp        = require('gulp');
var browserSync = require('browser-sync').create();
var sass        = require('gulp-sass');

// Static Server + watching scss/html files
gulp.task('serve', ['sass'], function() {

    browserSync.init({
        server: "./app"
    });

    gulp.watch("app/scss/*.scss", ['sass']);
    gulp.watch("app/*.html").on('change', browserSync.reload);
});

// Compile sass into CSS & auto-inject into browsers
gulp.task('sass', function() {
    return gulp.src("app/scss/*.scss")
        .pipe(sass())
        .pipe(gulp.dest("app/css"))
        .pipe(browserSync.stream());
});

gulp.task('default', ['serve']);
```

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.24.2  ]
- Node        [  10.0.0  ]
- Npm         [  6.0.0  ]

### Affected platforms

- [ ] linux
- [x] windows
- [ ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [x] Gulp
- [ ] Grunt
- [ ] CLI

### If CLI, please paste the entire command below

-

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

```
gulp.task('sass', () => {
  gulp
    .src('src/scss/main.scss')
    .pipe(ifElse(isDev, () => sourcemaps.init()))
    .pipe(sass().on('error', sass.logError))
    .pipe(postcss([autoprefixer()]))
    .pipe(ifElse(isDev, () => sourcemaps.write()))
    .pipe(ifElse(!isDev, () => cleanCSS()))
    .pipe(gulp.dest('dist'))
    .pipe(browserSync.stream());
});
```

**Comment by huilang-me** [comment]
Hope to fix it soon.

**Comment by preliot** [comment]
For everyone who can't wait, workaround. Use [http://livereload.com/](url) and [https://www.npmjs.com/package/gulp-livereload]. This works lighting fast!

**Comment by huilang-me** [comment]
We can simply use the old version to instead.
`npm uninstall browser-sync
npm install browser-sync@2.18.12
`

**Comment by shakyShane** [comment]
I'll be looking at this today

**Comment by shakyShane** [comment]
fixed in browser-sync@2.24.3
